src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,3 @@
-// 
+.card-image img {
```

Comment:
Here the same

Reply:
Sorry, I don't understand how I can change it. Or should I use BEM throughout the project?